MCprep_addon/spawner/spawn_util.py

```diff
@@ -41,11 +45,298 @@
+			('TAIGA', "Taiga", ""),
+		]
+		return items
+	
+	def zombie_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('DEFAULT', "Default", ""),
+			('DROWN', "Drown", ""),
+			('HUSK', "Husk", ""),
+		]
+		return items
+	
+	def skeleton_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('DEFAULT', "Default", ""),
+			('STRAY', "Stray", ""),
+			('WITHER', "Wither", ""),
+			# ('BOGGED', "Bogged", "") # 1.21 Added new skeleton varia
+		]
+		return items
+	
+	def axolotl_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('BLUE', "Blue", ""),
+			('CYAN', "Cyan", ""),
+			('GOLD', "Gold", ""),
+			('LUCY', "Lucy", ""),
+			('WILD', "Wild", ""),
+		]
+		return items
+	
+	def rabbit_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('BLACK', "Black", ""),
+			('BROWN', "Brown", ""),
+			('CAERBANNOG', "Caerbannog", ""),
+			('GOLD', "Gold", ""),
+			('SALT', "Salt", ""),
+			('TOAST',"Toast", ""),
+			('WHITE', "White", ""),
+			('WHITE_SPLOTCHED',"White splotched", ""),
+		]
+		return items
+	
+	def frog_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('COLD', "Cold", ""),
+			('TEMPERATE', "Temperate", ""),
+			('WARM', "Warm", ""),
+		]
+		return items
+	
+	def parrot_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('BLUE', "Blue", ""),
+			('GREEN',"Green", ""),
+			('GREY', "Grey", ""), 
+			('RED_BLUE', "Red blue", ""),
+			('YELLOW_BLUE', "Yellow blue", ""),
+		]
+		return items
+		
+	def llama_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('BROWN',"Brown", ""),
+			('CREAMY',"Creamy",""),
+			('GRAY', "Gray", ""),
+			('WHITE', "White", ""),
+		]
+		return items
+	
+	def horse_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('BLACK', "Black", ""),
+			('BROWN', "BROWN", ""),
+			('CHESTNUT', "CHESTNUT", ""),
+			('CREAMY', "CREAMY", ""),
+			('DARK_BROWN', "DARK_BROWN", ""),
+			('GRAY', "GRAY", ""),
+			('ZOMBIE', "ZOMBIE", ""),
+			('SKELETON', "SKELETON", ""),
+		]
+		return items
+		
+	def cat_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('ALL_BLACK', "All Black",""),
+			('BLACK', "Black", ""),
+			('BRITISH_SHORTHAIR', "British Shorthair", ""),
+			('CALICO', "Calico", ""),
+			('JELLIE', "Jellie", ""),
+			('OCELOT', "Ocelot", ""),
+			('PERSIAN', "Persian", ""),
+			('RAGDOLL', "Ragdoll", ""),
+			('RED', "Red", ""),
+			('SIAMESE', "Siamese", ""),
+			('TABBY', "Tabby", ""),
+			('WHITE', "White", ""),
+			# ('CAT_COLLAR', "", "")
+		]
+		return items
+	
+	def fox_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('NORMAL', "Normal", ""),
+			('SNOW', "Snow", "")
+		]
+		return items
+	
+	def squid_items(self, context: Context) -> List[Tuple[str, str, str]]:
+		items = [
+			('NORMAL', "Squid", ""),
+			('GLOW', "Glow", "")
+		]
```

Comment:
For example, with this, perhaps we could change `NORMAL` and `GLOW` to `BASE_MOB`/`NORMAL` and `ALTERNATE_MOB`, then use some properties like `MCPREP_MOB_BASE_MOB`/`MCPREP_MOD_NORMAL` and `MCPREP_MOB_ALTERNATE_MOB` to define what these states should be called in the UI.